lib/ssl-test.rb

```diff
@@ -1,12 +1,16 @@
+require "net/http"
 require "net/https"
+require "openssl"
+require "uri"
 
 module SSLTest
-  VERSION = "1.2.0"
+  VERSION = "1.2.0".freeze
```

Comment:
Let's bump to 1.3 for this
```suggestion
  VERSION = "1.3.0".freeze
```